### ISSUE TITLE
Add variant curved `phi` glyph for `U+03C6`

### ISSUE THREAD

**Issue by jebej** [microsoft/cascadia-code#588]
Currently, Cascadia code displays the same glyph (phi with straight vertical bar) for both `U+03D5` and `U+03C6`. It would be nice if the "curvy phi" glyph was added. Note that there exists confusion as to which glyph should match to which codepoint. For me personally, ideally it should match to latex/unicode-math:

Here's what it looks like in vscode with Cascadia:
![image](https://user-images.githubusercontent.com/11141558/138701770-935297fe-25d3-4960-a26e-1c0ababe38dc.png)

The above rendered correctly (in my opinion) by lualatex with [unicode-math package](https://ctan.org/pkg/unicode-math?lang=en):
![image](https://user-images.githubusercontent.com/11141558/138702009-ca5af3d6-093c-45b1-bc77-cbee4449bedc.png)

Raw text, which depending on browser font might or might not render correctly:
```latex
\begin{align}
  &\text{U+03D5 (phi):}    & \phi   \, ϕ \\
  &\text{U+03C6 (varphi):} & \varphi\, φ
\end{align}
```
Interestingly for me in the editor window the glyphs are correct but when rendered they are swapped (i.e. `U+03D5` gets the curvy symbol, wrong in my opinion).

**Comment by cormullion** [comment]
This phi stuff is quite confusing :), and I think the Unicode guidance isn't as clear (to me) as it could be:

<img width="1455" alt="Screenshot 2021-11-14 at 14 30 11" src="https://user-images.githubusercontent.com/52289/141685511-ffcb4960-e73a-4341-9adc-dbfc21d67266.png">

at http://unicode.org/reports/tr25/

But that last sentence:

> fonts that also intend to support technical use of the Greek letters should use the loopy form to contrast

is probably the reason why _coding_ fonts have generally adopted  a curved version of U+03C6, with Consolas being the exception that caused the confusion referred to above.

**Comment by jebej** [comment]
Thanks for this page. The explanation seems clear to me, it's just that it leaves the choice to implementers on whether to use the curved or straight glyph for `C6`.

With variants, as mentioned by @aaronbell, it is possible to support both a straight or curved `C6`, in addition to the always straight `D5`.

> _coding_ fonts have generally adopted a curved version of U+03C6

That would certainly be my preference, but I don't write Greek text! They do mention that the curved form is used more often in text too, though I can't comment on that.

> Consolas being the exception that caused the confusion referred to above.

That might be because they were following the earlier standard? It sure would be nice to get it fixed though...

**Comment by cormullion** [comment]
Yes, the OpenType Stylistic Set  or variations features work well in browser-based editors (VS-Code and Atom), it’s the terminal users who’d struggle, since many terminals and emulators are bad at OpenType support. ☹️
